cypress/locators/menu/locators.js

```diff
@@ -1,3 +1,3 @@
 // component preview locators
 export const MENU_PREVIEW = '[data-component="menu"]';
-export const SUBMENU_BLOCK = `${MENU_PREVIEW} > ul`;
+export const MENU_ITEM = 'div[data-component="menu-item"]';
```

Comment:
If is it possible, please remove `div`
```suggestion
export const MENU_ITEM = '[data-component="menu-item"]';
```

Reply:
deleted